spark/src/main/scala/geotrellis/spark/knn/KNearestRDD.scala

```diff
@@ -0,0 +1,27 @@
+package geotrellis.spark.knn
+
+import geotrellis.spark._
+import geotrellis.vector._
+import com.vividsolutions.jts.geom.Envelope
+import com.vividsolutions.jts.geom.Coordinate
+
+import org.apache.spark.rdd.RDD
+
+object KNearestRDD {
+  def kNearest[T](rdd: RDD[T], x: Double, y: Double, k: Int)(f: T => Extent): List[T] =
+    kNearest(rdd, new Envelope(new Coordinate(x, y)), k)(f)
```

Comment:
It actually looks like you're just creating an Extent, but using the implicit jts Envelope => Extent conversion to do so.

Should just use `Extent(x, y, x, y)`